README.md

```diff
@@ -43,7 +43,7 @@ If you are on Fedora, you can install the packaged version of the plugin by runn
 <a name="usage"></a>
 ## Usage
 
-The plugin is designed in an registration-manager-agnostic way which means that plugin itself does not depend on any OS nor way of registration. vagrant-registration only calls registration capabilities for given guest, passes the configuration options to them and handles interactive registration.
+The plugin is designed in a registration-manager-agnostic way which means that plugin itself neither depend on any OS nor way of registration. The vagrant-registration only calls registration capabilities for given guest, passes the configuration options to them and handles interactive registration.
 
```

Comment:
The plugin is designed in a registration-manager agnostic way, which means, that the plugin itself, depends neither on any OS nor on the way of registration. The vagrant-registration plugin only calls registration capabilities for the given guest, passes the configuration options to them and handles interactive registration.